### ISSUE TITLE
[UC]View EN and ZH version

### ISSUE THREAD

**Issue by lance-huang-nucloudglobal** [nucloudglobal/lms-ext#30]
This UC is created to separate the LMS language functions from other UC.
No FS document for this UC but client has confirmed:
1. LMS language needed: EN and ZH
2. Footer user same language as the other part of the LMS page

For Dev and SQA:
Client also agrees that we only need to do the general review for this UC in internal testing, UAT team at Conde Nast side will perform checking about this during Proc.UAT. Defects will be list as part of UAT issue list.

**Comment by jbaltazar** [comment]
@lance-huang-nucloudglobal 

I'm done for modifying some ( *Not all of them at the moment* ) text/verbiage to make multiple language from EN to ZH including the CRM Section for the following custom UC.

[UC-View student LMS profile #24](https://github.com/nucloudglobal/lms-ext/issues/24)
[UC-View student LMS profile #26](https://github.com/nucloudglobal/lms-ext/issues/26)
[UC-View student CRM profile #27](https://github.com/nucloudglobal/lms-ext/issues/27)

<b>Legend:</b>
When I say "<b>GUEST</b>", This is non authenticated user that is accessing or viewing non authenticated page of the website right now it's LMS.

<b>Current problem:</b>
For non authenticated page like "Login", we can't use the Http Broswer Info just to check what country that guest is belong where most of CN people is using VPN Connection (This is base from what we talk with @apetitnucloud ).

<b>Propose solution:</b>
Instead of detecting what country that guest is located via browser or via location of their ISP, We can

- Set the system to be CN language by default where LMS is for CN people
- Set the login page into CN Language by default same with ECW
- Add Language selection in non authenticated page like "Login Page". same in ECW under right top of the browser but this time for  LMS I suggest to put this after "LOGIN" button.
- If guest selected "English" as page language, The login page will be translated into EN but after the guest enter his/her account information (Username/Password) to be authenticated. Upon LMS redirection to Dashboard page. LMS will use the pre-define setting that User selected under [Settings Page](http://lmssqa.nucloudglobal.net/profile/settings). This is vice versa if guest selected the "ZH" and the LMS User predefined language settings is "English".

<b>Propose Layout for Login Page:</b>
![login_page](https://cloud.githubusercontent.com/assets/7844693/10991735/bd704838-849a-11e5-9596-78e46ee8ede8.png)

**Comment by lance-huang-nucloudglobal** [comment]
@jbaltazar 
Client confirmed:
1. Keep the system login page language same as now (login page languange is depending on what is the Defualt languange set in Conde Nast-> Settings)
2. if it is set to EN login page, show EN footer, if ZH, show ZH

CC @nucloud-agutierrez @apetitnucloud

**Comment by jbaltazar** [comment]
@lance-huang-nucloudglobal 

ok this is already done. I'm going to push the updated yml and setup so that @apetitnucloud can start reviewing and updating our server.

For this I just work for 1 day. where by default. Multiple language is not enable in my VM

**Comment by jbaltazar** [comment]
Footer is now dependent from what is the general default language base in [Client confirmed](https://github.com/nucloudglobal/lms-ext/issues/30#issuecomment-155271873).

[See technical information](https://github.com/nucloudglobal/lms/pull/8)

**Comment by jbaltazar** [comment]
@apetitnucloud is there any way we can see this changes live? This is 1 week old.

Note: This is not complicated code. Kindly let me know if there is unneeded thing that is needed to remove or modify.

CC: @lance-huang-nucloudglobal

**Comment by nucloud-agutierrez** [comment]
QA20160108: QA tested. Not Fixed.
- It shows EN login and footer even if the language was changed to ZH.

*see attached screenshot*

![screen shot 2016-01-08 at 12 29 53 pm](https://cloud.githubusercontent.com/assets/8049330/12190418/5008fb4c-b603-11e5-94bb-f2e5ed8e0134.png)

**Comment by lance-huang-nucloudglobal** [comment]
@jbaltazar @apetitnucloud 
I assume I did a right thing to remove the "Code Review" label which should have been removed by Aurelien with code is reviewed.

**Comment by jbaltazar** [comment]
@lance-huang-nucloudglobal 

Yes, you did the right thing. I'm going to label it again as "Code Review" if everything is already so that @apetitnucloud can review and integrate the changes in our server.

**Comment by jbaltazar** [comment]
@apetitnucloud 
Kindly check this pull request for the fix on footer width issue.
[Pull Request](https://github.com/nucloudglobal/lms/pull/13)

As for the Footer Language. Everything is working fine IF the "<b>Settings</b>" page under Condenast Account is working. Currently the "<b>Settings</b>" page is not working and it's return "<b>Page error</b>" so that SQA, Dev and PM can't change the System Language

See screenshot:
![systemsettingspage_error](https://cloud.githubusercontent.com/assets/7844693/12222280/d8961dbe-b7f2-11e5-9dfd-11f65200fb5e.png)


@nucloud-agutierrez 
Kindly check this small video clip on how to change the System Language over User Language
[Change_System_Language](https://www.dropbox.com/s/xv0fpv5ryqbwrmm/ChangeSystemLanguage.swf)

CC: @lance-huang-nucloudglobal @pvijayror

**Comment by nucloud-agutierrez** [comment]
QA20160113: QA tested. Not Fixed.
- Page Error is displayed upon accessing the settings page

*see attached screenshot*
![screen shot 2016-01-13 at 9 04 59 am](https://cloud.githubusercontent.com/assets/8049330/12281863/a9184b32-b9d4-11e5-88b6-d20f4e9bf549.png)

**Comment by lance-huang-nucloudglobal** [comment]
@apetitnucloud 
Can you pls fix this issue on env if it is not due to coding?

**Comment by jbaltazar** [comment]
@lance-huang-nucloudglobal 

Settings Page for changing System Language is now working. Kindlye check it everything is working fine.

Thank you

**Comment by nucloud-agutierrez** [comment]
QA20160114: QA tested. Not Fixed in Test Environment.
- As per client confirmed 
"2. if it is set to EN login page, show EN footer, if ZH, show ZH"

- It should show ZH footer since the default language selected is ZH.

*see attached screenshot*

![screen shot 2016-01-14 at 9 42 18 am](https://cloud.githubusercontent.com/assets/8049330/12313530/3cb83388-baa3-11e5-9c88-ee94d7bf687c.png)

**Comment by lance-huang-nucloudglobal** [comment]
@nucloud-agutierrez 
Hi Joy, pls review https://github.com/nucloudglobal/lms-ext/issues/30#issuecomment-155271873

Language of **login** page follows the Defualt languange set in Conde Nast-> Settings
After login, the language of the page follows users language setting which is the default LMS function

CC @jbaltazar

**Comment by nucloud-agutierrez** [comment]
QA20160116: QA tested. Fixed in Test Environment.

- Language of login page follows the Defualt languange set in Conde Nast -> Settings - *Passed*
- After login, the language of the page follows users language setting which is the default LMS function - *Passed*  (Settings on the upper left part of the page > Language)

**Comment by nucloud-agutierrez** [comment]
QA20150128: This UC is also fixed in PROD Environment.

QA20160116: QA tested. Fixed in Test Environment.

- Language of login page follows the Defualt languange set in Conde Nast -> Settings - *Passed*
- After login, the language of the page follows users language setting which is the default LMS function - *Passed*  (Settings on the upper left part of the page > Language)
